packages/web/config/ibc-assets.ts

```diff
@@ -868,6 +868,12 @@ export const IBCAssetInfos: (IBCAsset & {
     destChannelId: "channel-5",
     coinMinimalDenom: "stuatom",
   },
+  {
+    counterpartyChainId: "stride-1",
+    sourceChannelId: "channel-106",
```

Comment:
wrong src and dest channel Ids (these are for stargaze <-> stride), when it should be for (osmosis <-> stride)